test/remote/gateways/remote_orbital_test.rb

```diff
@@ -906,11 +906,33 @@ def test_void_transactions
   end
 
   def test_successful_verify
+    response = @gateway.verify(@credit_card, @options)
+    assert_success response
+    assert_equal 'No reason to decline', response.message
+  end
+
+  def test_successful_different_cards
+    cards = %w(visa master american_express diners_club jcb)
+    @credit_card.brand = cards.sample
```

Comment:
💬  Comment:

By using sample here you are adding randomness to the test, maybe is an edge case but let's say that jcb is failing for some reason, in that case, some tests execution are going to fail and others not.